src/Core/src/Handlers/FlyoutView/FlyoutViewHandler.cs

```diff
@@ -4,7 +4,7 @@
 using PlatformView = Android.Views.View;
 #elif WINDOWS
 using PlatformView = Microsoft.Maui.Platform.RootNavigationView;
-#elif NETSTANDARD || (NET6_0 && !IOS && !ANDROID)
+#elif (NETSTANDARD || !PLATFORM) || (NET6_0 && !IOS && !ANDROID)
```

Comment:
These are other weird cases because they define the same thing twice, but that is OK for this PR.